find-cursor.c

```diff
@@ -327,7 +345,7 @@ void draw(char *name, Display *display, int screen,
 		int cs = grow ? i : size - i;
 
 		if (outline) {
-			XSetLineAttributes(display, gc, line_width+2, LineSolid, CapButt, JoinBevel);
+			XSetLineAttributes(display, gc, line_width+6, LineSolid, CapButt, JoinBevel);
```

Comment:
Probably better to add a new `--outline-width`/`-W` flag for this; not everyone uses 4k displays, and the current value works well for others.